packages/eslint-config-jest/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * strvcom/eslint-config-jest
+ *
+ * @author      Michal Jarnot <m.jarnot@yahoo.com>
+ * @copyright   2019 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: ['plugin:jest-formatting/recommended'],
```

Comment:
```suggestion
  extends: ['plugin:jest-formatting/strict'],
```
Why not `strict`?

Reply:
Good point 🙂 ✅ 
![image](https://user-images.githubusercontent.com/14946081/81110350-ba420800-8f1b-11ea-90b2-0bf4b865b750.png)